project/MicroService.scala

```diff
@@ -34,41 +23,23 @@ trait MicroService {
       ScoverageKeys.coverageHighlighting := true
     )
   }
-
-
   lazy val microservice = Project(appName, file("."))
     .enablePlugins(Seq(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin) ++ plugins: _*)
```

Comment:
You don't need to pass all the plugins again. Change this to be only .enablePlugins(plugins : _*)